pyface/wizard/chained_wizard_controller.py

```diff
@@ -168,9 +168,10 @@ def _update(self):
 
     # Static ----
 
-    def _current_page_changed(self, old, new):
+    @observe("current_paage")
+    def _current_page_updated(self, event):
```

Comment:
```suggestion
    def _reset_observers_on_current_page_and_update(self, event):
```

i'm considering the `current_page` update on `self.next_controller` under the "update" part of the method name